src/windows/views/webview.py

```diff
@@ -1004,7 +995,7 @@ def Waveform_Ready(self, clip_id, audio_data):
         self.waveform_cache[clip_id] = serialized_audio_data
 
         # Pass to javascript timeline (and render)
-        self.run_js(JS_SCOPE_SELECTOR + ".setAudioData('" + clip_id + "', " + serialized_audio_data + ");")
+        self.run_js("drawAudio('" + clip_id + "');")
```

Comment:
Removing direct calls to `drawAudio`, since the Angular scope/watch will handle when to draw audio